.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+:tickets: [Jira ticket]()
+
+<!--
+Replace PATH_TO_FEATURE with Vercel link
+-->
+
+:technologist: [Admin preview](PATH_TO_FEATURE)
+
+:rose: [Rose preview](PATH_TO_FEATURE)
+
+:desktop_computer: [Desktop preview](PATH_TO_FEATURE)
+
+<!-- Add a brief description of changes here. Include any other necessary relevant links -->
```

Comment:
This totally personal and I will not ask for a change,m buit more for opinions:
- What do you think of moving the description comment right after the jira ticket?

Reply:
I actually thought the same thing but I wasn't sure if it would look weird before the description header. I will updated it to see how it looks and feels. Currently the JIRA ticket and the Vercel links are too close in my opinion.